README.md

```diff
@@ -8,8 +8,6 @@
 
 Reporting API client for Python
 
-TODO(cookiecutter): Improve the README file
-
```

Comment:
It's fine to improve the README later, but can you please create an issue for this.

Reply:
Opened this issue for improving Readme https://github.com/frequenz-floss/frequenz-client-reporting-python/issues/45